recursion/GCF.py

```diff
@@ -0,0 +1,7 @@
+def gcf(a, b):
+    """ calculate the GCF using the euclidean algorithm. """
+    if b == 0:
+        return abs(a)
+    return gcf(b, a%b)
```

Comment:
please add comments for a better understanding

Reply:
There's already a docstring explaining what algorithm the function is using, so someone can search that up if they're interested. Besides, it's just three lines of Python code, not much to explain.